www/src/pages/faq.md

```diff
@@ -4,11 +4,130 @@ pageTitle: FAQ
+- [Design Tokens Language Server](https://www.github.com/bennypowers/design-tokens-language-server)
+- [Figma](https://www.figma.com)
+- [Framer](https://www.framer.com)
+- [Knapsack](https://www.knapsack.cloud)
+- [Marvel](https://www.marvelapp.com)
+- [Penpot](https://www.penpot.com)
+- [Sketch](https://www.sketch.com)
+- [Style Dictionary](https://www.styledictionary.com)
+- [Superposition](https://www.superposition.design)
+- [Supernova](https://www.supernova.ion)
+- [Terrazzo](https://www.terrazzo.app)
+- [Toolabs](https://www.toolabs.com)
+- [TokensBrücke](https://www.figma.com/community/plugin/1254538877056388290)
+- [Tokens Studio](https://www.tokens.studio)
+- [Universal Design Tokens](https://www.github.com/universal-design-tokens/udt)
+- [Zeplin](https://www.zeplin.io)
+- [zeroheight](https://www.zeroheight.com)
+
+## How does this relate to other token approaches?
+
+The Design Tokens Community Group specification relates to other token approaches as a standardization layer that unifies previously fragmented, tool-specific formats. It provides:
+
+- **Canonical syntax** replacing incompatible proprietary formats
+- **Interoperability foundation** enabling cross-tool workflows
+- **Validation framework** ensuring format consistency
+- **Extension mechanism** preserving tool-specific capabilities
+
+The specification **complements rather than replaces** token methodologies and architectural patterns developed by production design systems. It defines the format for token exchange while leaving organizational strategy to design system teams.
+
+As of 2025, the specification approaches v1.0.0 release with growing adoption across major design tools and transformation pipelines. Teams should evaluate DTCG migration based on interoperability requirements, tooling support, and long-term maintenance considerations.
+
+## What is the relationship between JSON Schema and DTCG?
+
+JSON Schema is a declarative specification, a passive validator, and documentation format. It can be used to enforce documents to adhere to a strict format. It is a general purpose data interchange format.
+
+DTCG is a specification to provide indivisible pieces of a design system such as colors, spacing, typography scale. It is purpose-built for design token exchange. It is both a way to declare your design system token schema and a way to deliver the actual content.
+
+Similarities:
+
+- Both use JSON syntax (objects, strings, numbers, arrays, booleans, null)
+- Both support nested hierarchical structures
+- Both are human-readable and machine-parsable
+- Both use key-value pairs for data organization
+- Both can be validated against schemas
+
+While they may have some similarities, they have different purposes and features. DTCG has required properties, reserved keywords, type system and validation, standardized alias and reference system, type inheritance through groups, composite token structures, tooling and transformation expectations.
```

Comment:
This paragraph doesn’t make sense to me, because I don’t understand what “required properties” and “reserved keywords” mean, especially in contrast to JSON Schema. We could mirror the previous paragraph, like so:

```suggestion
Differences:

- DTCG serves as both a way to declare your schema _and_ exchange token data in the same file (JSON Schema is not for data exchange)
- JSON Schema is a generic tool; DTCG is tailored for design systems and design tokens
- DTCG types map to specific concepts in design tools and web and mobile app development
```

🤔 something along those lines